code/modules/vore/eating/vorepanel_vr.dm

```diff
@@ -611,7 +611,7 @@
 	var/atom/movable/target = locate(params["pick"])
 	if(!(target in host.vore_selected))
 		return TRUE // Not in our X anymore, update UI
-	var/list/available_options = list("Examine", "Eject", "Move")
+	var/list/available_options = list("Examine", "Eject", "Move", "Transfer")
 	if(ishuman(target))
 		available_options += "Transform"
 	intent = tgui_alert(user, "What would you like to do with [target]?", "Vore Pick", available_options, strict_byond = TRUE)
```

Comment:
A suggestion I'd have is using tgui_input_list instead of the alert one (which should actually swap to a list automatically though that was temporarily disabled), due to the list of buttons getting pretty long and sometimes overlapping stuff or stuff.

Otherwise it all looks fine! :D 

```suggestion
	intent = tgui_input_list(user, "What would you like to do with [target]?", "Vore Pick", available_options, strict_byond = TRUE)
```